openhcl/underhill_core/src/shutdown_relay.rs

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+//! Implements shutdown relay device for underhill.
```

Comment:
avoid using the term Underhill moving forwards in new code (even if the crate you're in has yet to be renamed)